src/main/java/com/greghaskins/spectrum/BDDSyntax.java

```diff
@@ -0,0 +1,69 @@
+package com.greghaskins.spectrum;
+
+import static com.greghaskins.spectrum.Spectrum.describe;
+import static com.greghaskins.spectrum.Spectrum.describeChain;
+import static com.greghaskins.spectrum.Spectrum.it;
+
+/**
+ * A translation from Spectrum describe/it to Gherkin like Feature/Scenario/Given/When/Then syntax
+ * Note - the beforeEach and afterEach will still be executed between given/when/then steps which
+ * may not make sense in many situations.
+ */
+public class BDDSyntax {
+	/**
+	 * Describes a feature of the system. A feature may have many scenarios.
+	 * @param featureName name of feature
+	 * @param block the contents of the feature
+	 */
+	public static void feature(final String featureName, final Block block) {
+		describe("Feature: " + featureName, block);
+	}
+
+	/**
+	 * Describes a scenario of the system. These can be at root level, though are best grouped
+	 * inside features.
+	 * @param scenarioName name of scenario
+	 * @param block the contents of the scenario - given/when/then steps
+	 */
+	public static void scenario(final String scenarioName, final Block block) {
+		describeChain("Scenario: " + scenarioName, block);
```

Comment:
Now uses a causal link within the steps to stop step running when there's a failure earlier in the list.